utils.js

```diff
@@ -0,0 +1,10 @@
+async function wrapWithTryCatch(res, callback) {
```

Comment:
one thing i observe here is, you using this function in every router call.
in call you are setting status 500 for every error.

Reply:
Yes, that can be customized. A new parameter is needed for code.